src/Runner.jl

```diff
@@ -420,6 +440,10 @@ function generate_compiler_wrappers!(platform::AbstractPlatform; bin_path::Abstr
         if lock_microarchitecture
             append!(flags, get_march_flags(arch(p), march(p), "gcc"))
         end
+        gcc_version, llvm_version = select_compiler_versions(p, compilers)
+        if libc(platform) == "musl" && gcc_version == v"4.8.5"
```

Comment:
I went through the output of `cc -E -v - < /dev/null` of all versions of GCC from 4 to 11, it looks like we need this also for GCC 5:
```suggestion
        if libc(platform) == "musl" && gcc_version in (v"4.8.5", v"5.2.0")
```